src/server.c

```diff
@@ -4089,7 +4089,7 @@ void addReplyCommand(client *c, struct redisCommand *cmd) {
             maplen++;
         }
         if (cmd->key_specs_num) {
-            addReplyBulkCString(c, "key-specs");
+            addReplyBulkCString(c, "key_specs");
```

Comment:
please mention these in the top comment.
does it make it a `major-decision`?
i suppose being a small "rename" on a non-released interface, it doesn't...

Reply:
I don't think it should be a `major-decision` (or maybe `major_decision`). Updated.